### PR TITLE
Added Airbnb eslint and prettier

### PR DESCRIPTION
**Pull request by dsiah**
See if you can install the dev dependencies and use them in intelliJ since these linting tools are supposed to be IDE agnostic.

Some major points about default linting:
- it defaults 2 space indents, thoughts?
- I still haven't figured out why but it doesn't like dangling commas
- spacing in braces is kinda contextual (react/jsx seems to opt against while destructuring or object literals are in favor)

[more details on airbnb style guide](https://github.com/airbnb/javascript)

[more details on react/jsx style guide](https://github.com/airbnb/javascript/tree/master/react)

### CODE REVIEW

File: src/Pad.js [hunk 1]
```diff
@@ -1,132 +1,141 @@
-import React, { Component } from 'react';
-import './Pad.css';
+import React, { Component } from "react";
```
Comment (dsiah) [inline src/Pad.js]: hmm I don't know why I swapped for double quotes, I'll see if I can turn that off

**Comment by maxcrane** [comment]
for spacing indents and trailing commas I don't really have a preference, I did get this warning message when npm installing on this branch

`npm WARN eslint-config-react-app@2.0.1 requires a peer of babel-eslint@^7.2.3 but none was installed.
npm WARN eslint-config-react-app@2.0.1 requires a peer of eslint-plugin-jsx-a11y@^5.1.1 but none was installed.
`

that's my only concern, but besides that I think this is fine to merge

**Comment by dsiah** [comment]
~Hmm I'll look into this warning I think I'll need to install babel-eslint~

After looking into this it seems as though its looking for older dependencies since we already have the specific deps in our devDependencies. We've got two choices, keep the versions we have or demote them to the specified ones. What do you think?

**Comment by maxcrane** [comment]
so create-react-app is using the older ones but airbnb linter wants newer ones? I'd say use the newer ones then

**Pull request closed without merging**